src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -13,6 +14,8 @@ namespace System.Diagnostics
     /// <summary>Base class used for all tests that need to spawn a remote process.</summary>
     public abstract partial class RemoteExecutorTestBase : FileCleanupTestBase
     {
+        private static bool outOfProc = (Environment.GetEnvironmentVariable("FORCE_LOCAL_INVOKE") != "1");
```

Comment:
Nit: readonly.  And s_ prefix.  I'd also have expected the name to better match the env var, e.g. s_forceLocalInvoke.